### ISSUE TITLE
Nodered docker crashes after short time after update to 4.0.0.

### ISSUE THREAD

**Issue by canedje** [node-red/node-red#4780]
### Current Behavior

After starting the docker it crashing after some time (couple of minutes)
LOG:
```
21 Jun 15:07:00 - [red] Uncaught Exception:
21 Jun 15:07:00 - [error] TypeError: Cannot read properties of undefined (reading 'includes')
    at init.<anonymous> (/data/node_modules/node-red-contrib-unifi-os/build/nodes/Protect.js:214:35)
    at Generator.next (<anonymous>)
    at /data/node_modules/node-red-contrib-unifi-os/build/nodes/Protect.js:31:71
    at new Promise (<anonymous>)
    at __awaiter (/data/node_modules/node-red-contrib-unifi-os/build/nodes/Protect.js:27:12)
    at Object.handleUpdate [as dataCallback] (/data/node_modules/node-red-contrib-unifi-os/build/nodes/Protect.js:139:40)
    at /data/node_modules/node-red-contrib-unifi-os/build/SharedProtectWebSocket.js:126:42
    at Array.forEach (<anonymous>)
    at WebSocket.<anonymous> (/data/node_modules/node-red-contrib-unifi-os/build/SharedProtectWebSocket.js:124:57)
    at WebSocket.emit (node:events:519:28)
    at Receiver.receiverOnMessage (/data/node_modules/ws/lib/websocket.js:1209:20)
    at Receiver.emit (node:events:519:28)
    at Receiver.dataMessage (/data/node_modules/ws/lib/receiver.js:567:14)
    at Receiver.getData (/data/node_modules/ws/lib/receiver.js:496:10)
    at Receiver.startLoop (/data/node_modules/ws/lib/receiver.js:167:16)
    at Receiver._write (/data/node_modules/ws/lib/receiver.js:94:10)
    at writeOrBuffer (node:internal/streams/writable:564:12)
    at _write (node:internal/streams/writable:493:10)
    at Writable.write (node:internal/streams/writable:502:10)
    at TLSSocket.socketOnData (/data/node_modules/ws/lib/websocket.js:1303:35)
    at TLSSocket.emit (node:events:519:28)
    at addChunk (node:internal/streams/readable:559:12)
```

### Expected Behavior

Just keep running as it did before

### Steps To Reproduce

Just start the docker

### Example flow

```
paste your flow here
```

### Environment

- Node-RED version: 4.0.0
- Node.js version:
- npm version:
- Platform/OS: Docker container under unraid
- Browser:

**Comment by knolleary** [comment]
This is not a crash coming from the core of Node-RED; it is likely an issue related to `node-red-contrib-unifi-os` - please raise an issue against that node: https://github.com/NRCHKB/node-red-contrib-unifi-os/issues

**Comment by GogoVega** [comment]
I also relayed the info internally 😉 (into NRCHKB)

**Comment by canedje** [comment]
Thanks for the answer. I did remove the unifi os pallet.
NR is now already running for 20 minutes without crashing

**Comment by Shaquu** [comment]
Woops

**Comment by Shaquu** [comment]
NRCHKB/node-red-contrib-unifi-os/issues/72

**Comment by marcus-j-davies** [comment]
This is issued is fixed (1.0.1)
